src/components/LinearProgressBar/LinearProgressBar.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { 
+  LinearProgressBar as CoreLinearProgressBar,
+  LinearProgressBarProps as CoreLinearProgressBarProps 
+  } from 'wix-ui-core/LinearProgressBar';
+import ToggleOn from 'wix-ui-icons-common/system/ToggleOn';
+import FormFieldError from 'wix-ui-icons-common/system/FormFieldError';
+import style from './LinearProgressBar.st.css';
+import { Tooltip } from '../Tooltip';
+import { bool } from 'prop-types';
+
+export interface LinearProgressBarProps extends CoreLinearProgressBarProps{
+  errorMessage?: string;
+  light?: boolean;
+}
+
+export const LinearProgressBar: React.SFC<LinearProgressBarProps> = (props: LinearProgressBarProps) => {
+
+  const { errorMessage, light, ...otherProps } = props;
+
+  return (
+    <CoreLinearProgressBar 
+      data-hook="progress-bar"
+      {...style('root', { light: light })}
```

Comment:
es6 short notation: `light: light` --> `light`

Reply:
just removed the props and forgot to refactor, my bad

Reply:
done